content/gateway_guide.md

```diff
@@ -506,46 +524,37 @@ You may also enable `RequireAuth` for your cold wallet in order to use the [Auth
 
 If ACME decides to use Authorized Accounts, ACME creates an interface for users to get their Ripple trust lines authorized by ACME's cold account. After Alice has extended a trust line to ACME from her Ripple account, she goes through the interface on ACME's website to require ACME authorize her trust line. ACME confirms that it has validated Alice's identity information, and then sends a TrustSet transaction to authorize Alice's trust line.
```

Comment:
This seems out of place here. Wouldn't it make more sense to move this paragraph to the beginning of the RequireAuth section? Then the structure of the "With Cold Wallets" and "With Hot Wallets" sections are parallel.